.circleci/config.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - run:
           name: Test
           command:
-            dotnet test -c Release --filter RunningMode!=Cluster
+            dotnet test -c Release --filter "RunningMode!=Cluster&Feature!=Analyzer"
```

Comment:
The Analyzers API is listed for 3.5 and 3.6 in the docs. Any reason for excluding the analyzer tests for those versions?

Reply:
Thanks for spotting this. I turned it off for these versions because they don't support the 'text' analyzer type, which I was using in my tests. I have changed the tests to user 'identifier' analyzer types and also re-enabled these two versions.